### ISSUE TITLE
[Extension Host] Error: AccessorVariable must have variable parent

### ISSUE THREAD

**Issue by leawp** [microsoft/vscode-js-debug#1218]
**Describe the bug**
Debugger disconnects, the debug console window stops being responsive

**To Reproduce**
It happens after some time but it's the nth time I got it today:
1. Leave the debbuger hanging on a breakpoint
2. enter following code into the debug console:

``` js 
Array.prototype.take = function(cnt){const arr = this.slice(0,cnt); return arr}
```

``` js 
Array.prototype.skip = function(cnt){const arr = this.slice(cnt); return arr}
```

``` js 
Set.prototype.toArray = function(){return Array.from(this)}
```

``` js 
Array.prototype.toSet = function(){return new Set(this)}
```

``` js
Object.prototype.toJson = function() {return JSON.stringify(this)}
```

use it like this (in the debug console)
``` js
someArray.filter(x=>x.someCondition === true)
    .skip(10) // skip first 10 results
    .take(20) // take only 20
    .map(x=>x.category) // pick or process field you want to research
    .toSet() // create a set of unique values
    .toArray() // get an array of those values

```

4. Go away and come back every now and then browsing through data local to the breakpoint
5. Change the filtering conditions
6. change the chain of functions a bit
7. vscode tab changes to Terminal from Debug Console
8. the app continues execution
9. debugger is left hanging on the breakpoint and unresponsive
10. See error in Window Log in vscode

**Log File**

```
[2022-03-07 12:14:08.290] [renderer3] [error] Error processing completions: Error: AccessorVariable must have variable parent
	at new H (c:\Users\...\app\extensions\ms-vscode.js-debug\src\extension.js:2:958513)
	at new G (c:\Users\...\app\extensions\ms-vscode.js-debug\src\extension.js:2:958880)
	at R.createVariable (c:\Users\...\app\extensions\ms-vscode.js-debug\src\extension.js:2:950189)
	at R.createPropertyVar (c:\Users\...\app\extensions\ms-vscode.js-debug\src\extension.js:2:953051)
	at R.createObjectPropertyVars (c:\Users\...\app\extensions\ms-vscode.js-debug\src\extension.js:2:951628)
	at processTicksAndRejections (internal/process/task_queues.js:93:5)
	at async z.getChildren (c:\Users\...\app\extensions\ms-vscode.js-debug\src\extension.js:2:959517)
	at async n.getVariables (c:\Users\...\app\extensions\ms-vscode.js-debug\src\extension.js:2:961962)
	at async c:\Users\...\app\extensions\ms-vscode.js-debug\src\extension.js:2:916410
	at async Promise.all (index 3)
	at async m.completions (c:\Users\...\app\extensions\ms-vscode.js-debug\src\extension.js:2:916513)
	at async f.defaultCompletions (c:\Users\...\app\extensions\ms-vscode.js-debug\src\extension.js:2:815063)
	at async f.identifierCompleter (c:\Users\...\app\extensions\ms-vscode.js-debug\src\extension.js:2:812821)
	at async T.completions (c:\Users\...\app\extensions\ms-vscode.js-debug\src\extension.js:2:928315)
	at async t.default._onMessage (c:\Users\...\app\extensions\ms-vscode.js-debug\src\extension.js:2:1063178): Error: Error processing completions: Error: AccessorVariable must have variable parent
```

vscode version 1.65.0

**Comment by connor4312** [comment]
Can you please collect a log file using the instructions in the issue template?

**Comment by leawp** [comment]
@connor4312 
I added trace:true (I was previously just running it from a JS debug terminal) but unfortunately the log file is about 130mb (gzipped) and its broken (there's an error message from 7zip saying "unexpected end of data" while unpacking). 

To add some context I'm browsing and analyzing a 84MB JSON file (its a dump from a relational database) through the Debug Console.

I think vscode might've ran into a memory limit.
After some more testing I've discovered a file in the dumps directory that wasn't gzipped, unfortunately it contains sensitive data and its not browsable (notepad++ takes up about 4.5GB of ram and then hangs) the file itself is about 1.8GB

...so I cannot send it over.
Sorry.

During the debugging session I've played around with data sometimes expanding the output of the commands that I've typed into the Debug Console. The data that was shown sometimes contained up to 65k rows in an array, I've mapped it, filtered it, printed it in many ways. I've also used autocomplete to get to properties of that loaded data.

I was running 2 instances of vscode and they were taking around 5.5GB of memory before the debugger's crash.
